### PR TITLE
README

### LINKED ISSUE #13: README
write the `README.md` which should include
- short summary of the project
- installation requirements
- how to run
- statement of contributions
- mention of `LICENSE.txt`
- contact info

### PR DESCRIPTION
**Pull request by medkamilmel**
Fixes #13.

Everyone in the group should approve this before merging.

### CODE REVIEW

**Review by ArnPellesGit — COMMENTED** [review]
The README looks nice to me!

Some small quirks/issues I've got:
* In the section running, perhaps you could add that "Go to your browser at localhost://<portnumber> to see what you're server's running." or something - it closes the loop imo.
* In the contributions section, could you add that I and Eldar did some bug-fixing/testing behavior together (through pairprogramming)?

File: README.md [hunk 1]
```diff
@@ -1,11 +1,103 @@
...
+<!-- CONTRIBUTIONS -->
+## Contributions
+
+Calin and Eldar worked together in setting up the environment and the server. 
+
+Eldar then implemented the model for the builds. 
+
+Per and Kamil implemented the web page through some pair programming sessions, with occasional help from Eldar in retrieving build data from the model. 
+
+Per and Eldar did some bug fixing/testing togethet through pair programming.
```
Comment (ArnPellesGit) [inline README.md:72]: Small typo: "togethet" -> "together"

Comment (medkamilmel) [reply README.md]: thanks for noticing, fixed it

**Review by ArnPellesGit — DISMISSED** [review]
LGTM!

**Review by capitanu — DISMISSED** [review]

**Comment by HakuPiku** [comment]
Please put a link to the website on readme.

**Comment by ArnPellesGit** [comment]
Add to the README that I wrote the Essence - work thingie

**Comment by ArnPellesGit** [comment]
We also need this in the README: "Preparation: the students document the build list URL in the README.", P7 to get pass with distinction in https://kth.instructure.com/courses/31884/assignments/185708

**Review by ArnPellesGit — APPROVED** [review]
LGTM

**Pull request merged**